src/primitives.jl

```diff
@@ -206,22 +206,6 @@ unwrapped `Nullable` entries.
 dropnull!(X::NullableVector) = deleteat!(X, find(X.isnull)).values # -> Vector
 
 """
-    anynull(X)
-
-Returns whether or not any entries of `X` are null.
-"""
-anynull(X::Any) = any(_isnull, X)           # -> Bool
-anynull(X::NullableArray) = any(X.isnull)   # -> Bool
```

Comment:
We still need to specialize `any(::typeof(isnull), X::NullableArray) = any(X.isnull)` for performance. Same for `all`.